webapp/src/main/resources/public/js/stores/PaginatorStore.js

```diff
@@ -34,7 +34,9 @@ class PaginatorStore {
     }
 
     changeCurrentPageNumber(currentPageNumber) {
-        this.currentPageNumber = currentPageNumber;
+        this.currentPageNumber = currentPageNumber
```

Comment:
Do you have any context why this change is needed (were we not displaying the page number in certain scenarios?)?

Reply:
Thanks for clarifying!